.github/workflows/code-quality.yml

```diff
@@ -10,7 +10,9 @@ jobs:
       with:
         python-version: "3.12"
     - run: pip install --upgrade pip
-    - run: pip install "ruff<1" "black<23" "isort<6" 
+
+    - run: pip install "ruff" "black" "isort"
+    
     - run: ruff check .
     - run: black --check .
```

Comment:
Add name for better logging for each step as installing dependencies and run each tool